custom-vu/src/main/kotlin/jces1209/vu/page/admin/issuetypes/BrowseIssueTypesPage.kt

```diff
@@ -0,0 +1,28 @@
+package jces1209.vu.page.admin.issuetypes
+
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import org.openqa.selenium.By
+import org.openqa.selenium.support.ui.ExpectedCondition
+import org.openqa.selenium.support.ui.ExpectedConditions
+
+abstract class BrowseIssueTypesPage(
+    private val jira: WebJira
+) {
+    protected val loadingPageExpectedCondition: ExpectedCondition<Boolean> = ExpectedConditions.and(
+        ExpectedConditions.visibilityOfElementLocated(By.className("aui-page-panel")),
+        ExpectedConditions.visibilityOfElementLocated(By.id("add-issue-type")),
+        ExpectedConditions.visibilityOfElementLocated(By.id("issue-types-table")),
+        ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[. = 'Bug']")),
+        ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[. = 'Epic']")),
+        ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[. = 'Story']")),
+        ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[. = 'Task']")),
+        ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[. = 'Sub-task']"))
```

Comment:
it is too general locators and is related to text
```suggestion
        presenceOfAllElementsLocatedBy(By.cssSelector("[data-issue-type-field]")),
        presenceOfAllElementsLocatedBy(By.cssSelector("[data-type]")),
        presenceOfAllElementsLocatedBy(By.className("operations-list")),
        visibilityOfElementLocated(By.className("search-entry"))
```